src/status_im2/contexts/quo_preview/profile/collectible.cljs

```diff
@@ -0,0 +1,55 @@
+(ns status-im2.contexts.quo-preview.profile.collectible
+  (:require
+   [quo2.components.profile.collectible.view :as quo]
+   [react-native.core :as rn]
+   [reagent.core :as reagent]
+   [status-im2.contexts.quo-preview.preview :as preview]))
+
+(def descriptor
+  [{:label   "Number of images"
+    :key     :num-imgs
+    :type    :select
+    :options (map (fn [n]
+                    {:key   n
+                     :value (str n " images")})
+                  (range 1 11))}
+   {:label "Random order of images"
+    :key   :shuffle-imgs
+    :type  :boolean}])
+
+(defn cool-preview
+  []
+  (let [state       (reagent/atom {:num-imgs 1 :shuffle-imgs false})
+        test-images ["https://i.seadn.io/gcs/files/ddefc5cbae2a8bc5d5aa368ca5887a23.jpg"
```

Comment:
oh its better to do not use external resources

Reply:
we can use same images used in designs and put them in mocks folder
https://github.com/status-im/status-mobile/tree/develop/resources/images/mock